contentcuration/automation/models.py

```diff
@@ -1,12 +1,19 @@
 import uuid
 
 from django.db import models
-from django.db.models import JSONField
+from kolibri_public.models import ContentNode
 
 
 class RecommendationsCache(models.Model):
     id = models.UUIDField(primary_key=True, default=uuid.uuid4, editable=False)
-    request = JSONField(default=dict, null=True)
-    response = JSONField(default=dict, null=True)
+    request_hash = models.CharField(max_length=32, null=True)
+    response = models.ForeignKey(
```

Comment:
Any reason not to make this `contentnode_id`?

Reply:
No particular reason, we can use `contentnode_id`. I guess its more descriptive.